### DISCUSSION TITLE
Combo input possible?

### DISCUSSION THREAD

**Discussion by coreservices-ron** [IBM/mac-ibm-notifications#213] [Q&A] [answered]
Dear Simone!

First of all, thank you for this perfect and powerful tool which gives us many options to interact from Shell scripts or even AppleScript! Highly appreciated!

I have a request though, which I am unsure if it's already built-in or would be a feature request: Is there an option to have an input area in combination with a drop down menu (maybe in a popup context)? I know this as „Combo Box“ from older UIs. So the user would be able to type any text into the input field, but could also select one option from the dropdown menu in order to quickly fill in a prepared text.

Is this currently implemented and I am only too blind to see how?

Thank you for any hint on this ...!

Kind regards,
  Ron.

**Comment by SMartorelli** [comment, marked as answer]
Hi @coreservices-ron, thanks!
The behaviour you described isn't currently covered by an IBM Notifier feature. You can have an `input` accessory view and a `dropdown` accessory view together on a popup but the two won't really interact between each other. Please feel free to [open a feature request](https://github.com/IBM/mac-ibm-notifications/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=%5BFeature+Request%5D) so that we can track it to plan work on it for the next release.
